ivc/src/ivc/mod.rs

```diff
@@ -91,8 +94,20 @@ mod tests {
         }
 
         println!("Building fixed selectors");
-        let fixed_selectors: Vec<Vec<Fp>> =
-            build_selectors::<_, TEST_N_COL_TOTAL, TEST_N_CHALS>(domain_size).to_vec();
+        let mut fixed_selectors: Vec<Vec<Fp>> =
+            build_selectors::<Fp, TEST_N_COL_TOTAL, TEST_N_CHALS>(domain_size).to_vec();
+
+        // Write constants
+        {
+            let rc = PoseidonBN254Parameters.constants();
```

Comment:
Any reason this block that's repeated twice cannot be a separate function `build_selectors() -> Vec<Vec<Fp>>`?

Reply:
Interpreter depends on the field.